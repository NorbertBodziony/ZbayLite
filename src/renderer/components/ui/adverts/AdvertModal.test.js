/* eslint import/first: 0 */
import React from 'react'
import { shallow } from 'enzyme'
import BigNumber from 'bignumber.js'

import { AdvertModal } from './AdvertModal'
import { mockClasses } from '../../../../shared/testing/mocks'

describe('AdvertModal', () => {
  it('renders component AdvertModal', () => {
    const result = shallow(
      <AdvertModal
        classes={mockClasses}
        balanceZec={new BigNumber(0.7)}
        rateUsd={new BigNumber(0.7)}
        rateZec={1}
        handleClose={jest.fn()}
        handleSend={jest.fn()}
        setFieldValue={jest.fn()}
        open
        isValid
        values={{}}
        touched={{}}
        errors={{}}
        submitForm={jest.fn()}
        sending={false}
      />
    )
    expect(result).toMatchSnapshot()
  })
})
