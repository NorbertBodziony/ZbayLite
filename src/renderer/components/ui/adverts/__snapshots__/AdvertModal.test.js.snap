// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdvertModal renders component AdvertModal 1`] = `
<Memo(WithStyles(Modal))
  alignCloseLeft={true}
  classes={
    Object {
      "window": "window-mock",
    }
  }
  fullPage={true}
  handleClose={[MockFunction]}
  open={true}
  title="New Listing"
>
  <AutoSizer
    disableHeight={false}
    disableWidth={false}
    onResize={[Function]}
    style={Object {}}
  >
    <Component />
  </AutoSizer>
</Memo(WithStyles(Modal))>
`;
